README.md

```diff
@@ -8,6 +8,19 @@ tutorials first and then practice these codes on your system.
 You can visit https://www.youtube.com/c/TheEasyConcepts/ for
 watching all the tutorials of C++.
 
+one information for the welfare of students: https://hacktoberfest.digitalocean.com/profile  ,this is the websites where you can get t-shirts in free;
+steps for getting free t-shirt:
+1.click on the above link or type Hacktoberfest in google.
+2.click on start hacking.
+3.connect your Github account by simply clicking on Auhtorize hacktoberfest.
+4.enter the data which is being asked by the form.
+5.then you have to go for 4 pull request.
+6.If your pull request reaches to 4 ,they will send u t-shirt at your home at free of cost.
+
+
+This fest happens every year in october.In 2020 it's happening in 1st of october..
+
+
```

Comment:
Please add your lines in between two long horizontal dashed lines.
-------------------------------------------------------------------------------------------------------------------
-------------------------------------------------------------------------------------------------------------------
Your Lines Here. Focus on formatting your lines so that instructions are clearly visible.
Write your lines just before this line - "Please do contact me in case of any query."
-------------------------------------------------------------------------------------------------------------------
-------------------------------------------------------------------------------------------------------------------